### ISSUE TITLE
This package is currently not on melpa yet

### ISSUE THREAD

**Issue by nick-youngblut** [Emiller88/nextflow-mode#10]
> This package is currently not on melpa yet

...and it appears that this repo hasn't been altered in >1 year, so I'm guessing that this package will likely never be on melpa. Is that a fairly safe assumption?

**Comment by nick-youngblut** [comment]
I got this running via the following:

```
;; straight.el setup
(defvar bootstrap-version)
(let ((bootstrap-file
       (expand-file-name "straight/repos/straight.el/bootstrap.el" user-emacs-directory))
      (bootstrap-version 5))
  (unless (file-exists-p bootstrap-file)
    (with-current-buffer
        (url-retrieve-synchronously
         "https://raw.githubusercontent.com/raxod502/straight.el/develop/install.el"
         'silent 'inhibit-cookies)
      (goto-char (point-max))
      (eval-print-last-sexp)))
  (load bootstrap-file nil 'nomessage))

;; groovy
(straight-use-package 'groovy-mode)

;; nextflow
(straight-use-package '(nextflow-mode :type git :host github :repo "Emiller88/nextflow-mode"))
(require 'nextflow-mode)
```

**Comment by edmundmiller** [comment]
> > This package is currently not on melpa yet
> 
> ...and it appears that this repo hasn't been altered in >1 year, so I'm guessing that this package will likely never be on melpa. Is that a fairly safe assumption?

Nah, I just haven't gotten around to making the PR to melpa :stuck_out_tongue: . I'll get it added. I haven't really needed anything new, so I haven't added anything. I'd love to add some other functionality if anyone has any thoughts, the only thing I want is `imenu` support to work with DSL2. I could probably update some DSL2 stuff now that the dust seems to have settled...

**Comment by edmundmiller** [comment]
https://github.com/melpa/melpa/pull/8163

**Comment by pwaltman** [comment]
> I got this running via the following:
> 
> ```
> ;; straight.el setup
> (defvar bootstrap-version)
> (let ((bootstrap-file
>        (expand-file-name "straight/repos/straight.el/bootstrap.el" user-emacs-directory))
>       (bootstrap-version 5))
>   (unless (file-exists-p bootstrap-file)
>     (with-current-buffer
>         (url-retrieve-synchronously
>          "https://raw.githubusercontent.com/raxod502/straight.el/develop/install.el"
>          'silent 'inhibit-cookies)
>       (goto-char (point-max))
>       (eval-print-last-sexp)))
>   (load bootstrap-file nil 'nomessage))
> 
> ;; groovy
> (straight-use-package 'groovy-mode)
> 
> ;; nextflow
> (straight-use-package '(nextflow-mode :type git :host github :repo "Emiller88/nextflow-mode"))
> (require 'nextflow-mode)
> ```

What file did you add this code to in order to get this running? I've tried modifying the <DOOMDIR>/packages.el (and ran `doom sync`, but it appears that it hasn't installed.

**Comment by edmundmiller** [comment]
On doom it should just be 
```elisp
(package! nextflow-mode :recipe (:host github :repo "emiller88/nextflow-mode"))
```
In your `packages.el`
and
```elisp
(use-package! nextflow-mode
  :config
  (set-docsets! 'nextflow-mode "Groovy"))
```

**Comment by progfolio** [comment]
@Emiller88 any interest in finishing off that MELPA pull request?

**Comment by edmundmiller** [comment]
Hey @progfolio! Honestly very little, I think with straight, elpaca, etc. It's pretty easy to install from a repo nowadays so I haven't had many people asking for it. 

If there's enough interest I can look at it after the Nextflow summit in December!

**Comment by progfolio** [comment]
Okay. I was asking because I was going through MELPA submissions which have not received any response in some time. Please consider communicating your intention on the  MELPA issue. Thanks.

**Comment by edmundmiller** [comment]
😆 Missed who you were. Happy to take the conversation on that PR!
